src/OpenLoco/src/Audio/Audio.cpp

```diff
@@ -1027,17 +1028,18 @@ namespace OpenLoco::Audio
         }
     }
 
-    static void addAllSongsToPlaylist(std::vector<uint8_t>& playlist)
+    static std::vector<uint8_t> makeAllMusicPlaylist()
     {
-        for (auto i = 0; i < kNumMusicTracks; i++)
-        {
-            playlist.push_back(i);
-        }
+        std::vector<uint8_t> playlist(kNumMusicTracks);
+        std::iota(playlist.begin(), playlist.end(), 0);
+        return playlist;
     }
 
-    static void addCurrentEraSongsToPlaylist(std::vector<uint8_t>& playlist)
+    static std::vector<uint8_t> makeCurrentEraPlaylist()
     {
+        std::vector<uint8_t> playlist = std::vector<uint8_t>();
```

Comment:
This is quite verbose, pick one of the following:
```suggestion
        auto playlist = std::vector<uint8_t>();
```
or
```suggestion
        std::vector<uint8_t> playlist;
```